test/katex-spec.js

```diff
@@ -2893,7 +2893,7 @@ describe("Unicode", function() {
     });
 
     it("should parse symbols", function() {
-        expect("£¥ðℂℍℑℓℕ℘ℙℚℜℝℤℲℵℶℷℸ⅁∀∁∂∃∇∞∠∡∢♠♡♢♣♭♮♯✓°\u00b7").toParse();
+        expect("£¥ðℂℍℑℓℕ℘ℙℚℜℝℤℲℵℶℷℸ⅁∀∁∂∃∇∞∠∡∢♠♡♢♣♭♮♯✓°¬‼\u00b7").toParse();
```

Comment:
It occurs to me that these tests should be done in strict mode.

Reply:
@edemaine  The same thing occurred to me.

Reply:
@ronkok Do you want to take a crack at this?  There are probably many of these tests.  You should just need to replace `toParse()` or `toBuild()` with `toParse(strictSettings)` or `toBuild(strictSettings)`.

Reply:
Sure. I can get to it later this week.